### PR TITLE
Adding support for git branch delete method in fish

### PR DESCRIPTION
**Pull request by cjappl**
Added `gbd` in fish. 

I often find that I end up with a bunch of stale branches, and I end up deleting them by hand. 

This allows for a nice way to easily delete my old branches, in a `forgit` complient way.

**NEEDS DUPLICATION IN BASH BY @wfxr or others :) **

## Check list

- [x] I have performed a self-review of my code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Type of change

- [x] New feature

## Test environment

- Shell
    - [x] fish
- OS
    - [x] Mac OS X

### CODE REVIEW

File: conf.d/forgit.plugin.fish [hunk 1]
```diff
@@ -237,6 +237,31 @@ function forgit::checkout::commit -d "git checkout commit selector" --argument-n
...
+    set preview "git log {1} --graph --pretty=format:'$forgit_log_format' --color=always --abbrev-commit --date=relative"
+
+    set opts "
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s --multi --tiebreak=index --header-lines=1
+        --preview=\"$preview\"
+        $FORGIT_BRANCH_DELETE_FZF_OPTS
+        "
+
+    set cmd "git branch --color=always --verbose --all | sort -k1.1,1.1 -r"
```
Comment (wfxr) [inline conf.d/forgit.plugin.fish:252]: Apply changes intruduced in #196?

Comment (cjappl) [reply conf.d/forgit.plugin.fish]: Done!

File: conf.d/forgit.plugin.fish [hunk 2]
```diff
@@ -237,6 +237,31 @@ function forgit::checkout::commit -d "git checkout commit selector" --argument-n
...
+    set preview "git log {1} --graph --pretty=format:'$forgit_log_format' --color=always --abbrev-commit --date=relative"
+
+    set opts "
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s --multi --tiebreak=index --header-lines=1
+        --preview=\"$preview\"
+        $FORGIT_BRANCH_DELETE_FZF_OPTS
+        "
+
+    set cmd "git branch --color=always --all | LC_ALL=C sort -k1.1,1.1 -rs"
```
Comment (wfxr) [inline conf.d/forgit.plugin.fish:252]: `--all` flag should be removed

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**